.editorconfig

```diff
@@ -0,0 +1,42 @@
+root = true
+
+[*]
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
+# indent_style = space
+# indent_size = 4
```

Comment:
can you enable this, so everyone will see same tab size

Reply:
I think we should leave default for language specific config